html/canvas/element/text/2d.text.measure.actualBoundingBox.html

```diff
@@ -43,6 +43,12 @@ <h1>2d.text.measure.actualBoundingBox</h1>
   _assert(ctx.measureText('A').actualBoundingBoxAscent >= 35, "ctx.measureText('A').actualBoundingBoxAscent >= 35");
   _assert(Math.abs(ctx.measureText('A').actualBoundingBoxDescent) <= 1, "Math.abs(ctx.measureText('A').actualBoundingBoxDescent) <= 1");
 
+  _assert(Math.abs(ctx.measureText('A ').actualBoundingBoxLeft) <= 1, "Math.abs(ctx.measureText('A ').actualBoundingBoxLeft) <= 1");
+  _assert(ctx.measureText('A ').actualBoundingBoxRight >= 50, "ctx.measureText('A ').actualBoundingBoxRight >= 50");
+
+  _assert(Math.abs(ctx.measureText(' A').actualBoundingBoxLeft) >= 50, "Math.abs(ctx.measureText(' A').actualBoundingBoxLeft) >= 50");
+  _assert(ctx.measureText(' A').actualBoundingBoxRight <= 101, "ctx.measureText(' A').actualBoundingBoxRight <= 101");
```

Comment:
It might be useful to separate this out so it can be tested standalone. I think we also want to test some other whitespace code points. Unicode has a lot of them.